src/findNodeHandle.web.ts

```diff
@@ -1,29 +1,31 @@
-type GestureHandlerRef = {
-  viewTag: GestureHandlerRef;
-  current: HTMLElement;
-};
+import type { GestureHandlerRef, SVGRef } from './web/interfaces';
+import { isSVGElement } from './web/utils';
 
 export default function findNodeHandle(
-  viewRef: GestureHandlerRef | HTMLElement
-): HTMLElement | number {
+  viewRef: GestureHandlerRef | SVGRef | HTMLElement | SVGElement
+): HTMLElement | SVGElement | number {
   // Old API assumes that child handler is HTMLElement.
   // However, if we nest handlers, we will get ref to another handler.
   // In that case, we want to recursively call findNodeHandle with new handler viewTag (which can also be ref to another handler).
   if ((viewRef as GestureHandlerRef)?.viewTag !== undefined) {
     return findNodeHandle((viewRef as GestureHandlerRef).viewTag);
   }
 
-  if (viewRef instanceof HTMLElement) {
+  if (viewRef instanceof HTMLElement || viewRef instanceof SVGElement) {
```

Comment:
Can't we check simply for `Element`?

Reply:
Seems like we can, both of them extend `Element`. Moreover, `MathMLElement` also  does this 😅 

Done in [dfe53b6](https://github.com/software-mansion/react-native-gesture-handler/pull/3197/commits/dfe53b6ec3a2409b6655ab329a2d51d52170f4df)